src/pages/colinks/CoLinksTaskCards.tsx

```diff
@@ -0,0 +1,164 @@
+import { QUERY_KEY_COLINKS } from 'features/colinks/wizard/CoLinksWizard';
+import { client } from 'lib/gql/client';
+import { useQuery } from 'react-query';
+import { NavLink } from 'react-router-dom';
+
+import { coLinksPaths } from 'routes/paths';
+import { Button, Flex, Panel, Text } from 'ui';
+
+export const CoLinksTaskCards = ({
+  currentUserAddress,
+}: {
+  currentUserAddress: string;
+}) => {
+  const { data: myProfile } = useQuery(
+    [QUERY_KEY_COLINKS, currentUserAddress, 'taskRep'],
```

Comment:
if i buy a link, does this cache get invalidated automatically?

usually the variable in query key is at the end, but it depends on exactly how we want to invalidate and refresh these i guess: https://tanstack.com/query/v4/docs/react/guides/query-keys @wingfeatherwave

Reply:
seems to be working fine, so appriving